asyncspotify/audioanalysis.py

```diff
@@ -0,0 +1,22 @@
+from .utils	import convertToTimeDelta
+from .object import Object
+
+class AudioAnalysis(Object):
+	'''
+	Class for Audio Analysis results from the Spotify API.
+    
+	'''
+	_type = 'audio_analysis'
+
+	def __init__(self, client, data):
+		super().__init__(client, data)
+		# List of seconds fields in data
+		listOfFieldsToConvert = ['start','duration','offset_seconds','window_seconds','start_of_fade_out','loudness_max_time']
+		data = convertToTimeDelta(data, listOfFieldsToConvert)
```

Comment:
I think it's better to explicitly instantiate each attribute here in the constructor instead of calling a spin-off function to do that work, especially since this is the only model that has a large amount of timedeltas

Reply:
I've reimplemented this using for loops for each of the lists. There's a bit of code duplication,
but is tidier now